library/src/main/java/com/datatorrent/lib/io/PubSubWebSocketAppDataQuery.java

```diff
@@ -105,11 +105,11 @@ public void endWindow()
   @Override
   public void teardown()
   {
+    super.teardown();
```

Comment:
Is it needed to be before BoundedService teardown? Shouldn't the teardown of operator be after service termination?